### PR TITLE
review web page

### PR DESCRIPTION
**Pull request by Vovan707**

### CODE REVIEW

**Comment by Vovan707** [comment]
@sergeyradkevich  sorry,I would rename the old brunch, but because of incorrect command in the console it deleted ((

**Comment by sergeyradkevich** [comment]
Ok, let me know when it is ready to review, because [this comment](https://github.com/Vovan707/Favorite-Places/pull/1#discussion_r26469913) is not worked out yet.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      <hr>
+      <div id = "lviv">
+        <img src="images/wall2.jpg " width="920px">
+      </div>
+      <div id = "kruivka">
+        <img src="images/Kryivka.jpg" height="230px">
+        <aside>
+          <section>
+            <p>
+              «<span class = "place-name">Криї́вка</span>» — ресторан-кнайпа у Львові. Відкритий у вересні 2007 р. Заклад оздоблений у вигляді польової криївки УПА, знаходиться у підвалі одного з будинків за адресою площа Ринок, 14 (Кам'яниця Венеційська). Кухня — українська, польова. Страви подають у військовому посуді. У ресторані є такі атракції, як тир та вільне користування зброєю. Традиція входу до закладу передбачає знання гасла-пароля Слава Україні - Героям слава, не назвавши який не можна потрапити всередину та перевірку чаркою медовухи для тих, хто старший 21 року.
```
Comment (sergeyradkevich) [inline index.html]: @Vovan707 Put `«`and `»` inside `span` around the value.

**Comment by sergeyradkevich** [comment]
Header line has wrong color

**Comment by Vovan707** [comment]
@sergeyradkevich  review please

**Comment by sergeyradkevich** [comment]
@Vovan707 Let me mention 3d time that [the following code](https://github.com/Vovan707/Favorite-Places/blob/create-web-page/index.html#L45-L56) is not formatted right. Leading indents are not the proper ones.

**Comment by Vovan707** [comment]
@sergeyradkevich soory, I formatting now it

**Comment by Vovan707** [comment]
@sergeyradkevich review please

**Comment by sergeyradkevich** [comment]
Wrong formatting

**Comment by Vovan707** [comment]
@sergeyradkevich  review please

**Pull request merged**